src/Compilers/CSharp/Portable/Symbols/Source/ExtensionGroupingInfo.cs

```diff
@@ -37,8 +37,7 @@ public ExtensionGroupingInfo(SourceMemberContainerTypeSymbol container)
                     continue;
                 }
 
-                var sourceNamedType = (SourceNamedTypeSymbol)type;
```

Comment:
>var sourceNamedType = (SourceNamedTypeSymbol)type; [](http://example.com/codeflow?start=16&length=50)

My recommendation is to revert changes in this file.

Reply:
Then we need to add assertions for nullability